src/Components/AnimateOnReveal.js

```diff
@@ -0,0 +1,29 @@
+import * as React from "react";
+import Zoom from "react-reveal/Zoom";
+import Fade from "react-reveal/Fade";
+
+function AnimateOnReveal({ animation, children }) {
```

Comment:
I did think again about the implementation of `AnimateOnReveal`. My next iteration is now this:

```js
import * as React from "react";
import Zoom from "react-reveal/Zoom";
import Fade from "react-reveal/Fade";

function AnimateOnReveal({ animation, children }) {
  switch (animation) {
    case "zoomIn":
      return <Zoom>{children}</Zoom>;
    case "fadeInLeft":
      return <Fade left>{children}</Fade>;
    case "fadeInRight":
      return <Fade right>{children}</Fade>;
    case "fadeInDown":
      return <Fade top>{children}</Fade>;
    case "fadeInUp":
      return <Fade bottom>{children}</Fade>;
    default:
      return children;
  }
}

export default AnimateOnReveal;
```

I think it is a bit clearer to read and also has a better default - rendering the children. E.g. if someone gives a not yet supported animation prop (e.g. "none") the code just works.